src/Inventory/Resources/templates/line-item.twig

```diff
@@ -0,0 +1,170 @@
+									<option value="PRICE_PRIORITY" selected>Price priority</option>
+									<option value="HOUSE">House</option>
+								</select>
+							</div>
+							<div class="form-group type-level" style="display: none;">
+								<select class="form-control" id="typeLevel" name="typeLevel">
+									<option value="HIGH">High</option>
+									<option value="NORMAL" selected>Normal</option>
+									<option value="LOW">Low</option>
+								</select>
+							</div>
+							<div class="form-group">
+								<input type="text" class="form-control" id="priority" placeholder="Priority" name="priority" />
+							</div>
+						</div>
+						<div class="form-group">
+							<div class="form-group">
+								<label for="start">Start time (UTC)</label>
+								<input type="text" class="form-control" id="start" name="start" placeholder="Leave empty for immediately run" />
+							</div>
+						</div>
+						<div class="form-group">
+							<div class="form-group">
+								<label for="start">End time (UTC)</label>
+								<input type="text" class="form-control" id="end" name="end" placeholder="Leave empty for unlimited" />
+							</div>
+						</div>
+						<div class="form-group">
+							<div class="form-group">
+								<label for="rate">Rate (USD CPM)</label>
+								<input type="text" class="form-control" id="rate" name="rate" placeholder="9.00" />
+							</div>
+						</div>
+						<hr />
+						<div class="form-group">
+							<div class="form-group">
+								<label for="rate">Key-values</label>
+							</div>
+						</div>
+						<div class="form-group key-value-pairs">
+							<div class="row text-right form-group add-pair-container">
+								<div class="btn btn-wikia add-pair"><span class="glyphicon glyphicon-plus" aria-hidden="true"></span> ADD PAIR</div>
+							</div>
+						</div>
+						<hr />
+						<div class="row text-right">
+							<button class="btn btn-wikia"><span class="glyphicon glyphicon-ok" aria-hidden="true"></span> SUBMIT LINE ITEM</button>
+						</div>
+					</div>
+				</form>
+			</div>
+		</div>
+		<script id="pair-template" type="text/template">
+			<div class="row key-value-pair">
+				<div class="form-group col-xs-4">
+					<input type="text" class="form-control key" name="keys[]" placeholder="e.g. src" />
+				</div>
+				<div class="form-group col-xs-2">
+					<select class="form-control operator" name="operators[]">
+						<option value="IS" selected>is</option>
+						<option value="IS_NOT">is not</option>
+					</select>
+				</div>
+				<div class="form-group col-xs-5">
+					<input type="text" class="form-control value" name="values[]" placeholder="e.g. gpt, mobile" />
+				</div>
+				<div class="col-xs-1">
+					<div class="btn btn-wikia remove-pair"><span class="glyphicon glyphicon-trash" aria-hidden="true"></span></div>
+				</div>
+			</div>
+		</script>
+		<script src="{{ getResourcePath('Common:js/jquery.min.js') }}" type="text/javascript"></script>
```

Comment:
What do you think of placing common scripts in a sub-template which we'll include here and in `tableau-web-connector.twig`?